### DISCUSSION TITLE
Server ownership label in the app

### DISCUSSION THREAD

**Discussion by rascarlo** [mullvad/mullvadvpn-app#8950] [Feature requests & Ideas]
Can we get a label with the server ownership in the long list of the servers? That would be a nice little detail to have.
thanks

**Comment by ProgressiveArchitect** [comment]
I second this.

I think it would also be a good idea to include whether the server is Owned or Rented by Mullvad. This type of information is important to users who have restrictive threat models related to provider trust.

**Comment by rascarlo** [comment]
any reply from the team?

**Comment by faern** [comment]
This is in the backlog but has not been implemented yet. We very recently changed what internal API we use to one that provides this info to the app, so we are one step closer to being able to add it at least.

**Comment by rascarlo** [comment]
> This is in the backlog but has not been implemented yet. We very recently changed what internal API we use to one that provides this info to the app, so we are one step closer to being able to add it at least.

Great, thanks!

**Comment by inson1** [comment]
It would be great feature
